contracts/src/PriceFeed.sol

```diff
@@ -20,31 +20,31 @@ contract PriceFeed is IPriceFeed {
     }
 
     /// @dev returns the usdc value of 1 mio DEGEN
-    function getPrice() external view returns (uint256) {
-        return degenToUsdc(1e6 * 1e18);
+    function getPrice(uint128 secondsAgo_) external view returns (uint256) {
+        return degenToUsdc(1e6 * 1e18, secondsAgo_);
     }
 
-    function degenToUsdc(uint128 degenAmount_) public view returns (uint256) {
-        uint256 quoteEth = degenToEth(degenAmount_);
-        uint256 quoteUsdc = ethToUsdc(uint128(quoteEth));
+    function degenToUsdc(uint128 degenAmount_, uint128 secondsAgo_) public view returns (uint256) {
+        uint256 quoteEth = degenToEth(degenAmount_, secondsAgo_);
+        uint256 quoteUsdc = ethToUsdc(uint128(quoteEth), secondsAgo_);
         return quoteUsdc;
     }
 
-    function degenToEth(uint128 degenAmount_) public view returns (uint256) {
+    function degenToEth(uint128 degenAmount_, uint128 secondsAgo_) public view returns (uint256) {
         uint32[] memory secondsAgos = new uint32[](2);
-        secondsAgos[0] = 0;
-        secondsAgos[1] = 60;
+        secondsAgos[0] = secondsAgo_;
+        secondsAgos[1] = secondsAgo_ + 60;
         (int56[] memory tickCumulatives,) = ethDegenPool.observe(secondsAgos);
         int56 tickCumulative = tickCumulatives[1] - tickCumulatives[0];
```

Comment:
I suspect these are backwards since tickCumulatives[1] is farther in the past then tickCumulatives[2], unless I am missing something